cmd/argocd/commands/app.go

```diff
@@ -759,6 +759,7 @@ func NewApplicationDiffCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 					}
 
 					printDiff(item.key.Name, target, live)
+					os.Exit(1)
```

Comment:
Sorry didn't catch this earlier. Can we exit after we print all the diffs, instead of bailing on the first one?

Reply:
fixed!